src/components/Toggle/Toggle.js

```diff
@@ -112,13 +124,18 @@ Toggle.propTypes = {
   /**
    * Removes the default bottom margin from the input.
    */
-  noMargin: PropTypes.bool
+  noMargin: PropTypes.bool,
+  /**
+   * Adds the ability of the component to be right-aligned.
+   */
+  rtl: PropTypes.bool
```

Comment:
The abbreviation "RTL" is generally associated with right-to-left languages such as Arabic or Hebrew and I would like to reserve the `rtl` prop name for this future use-case. For the same reason, the prop name should not include `right` or `left` since it would be the opposite in RTL language contexts. How about `flipped` or `reversed`? 

```suggestion
  /**
   * Reverses the order or the label and switch.
   */
  reversed: PropTypes.bool
```

Reply:
Agreed, _reversed_ makes much more sense.